WalletWasabi.Fluent/Converters/MoneyConverters.cs

```diff
@@ -8,4 +8,7 @@ public static class MoneyConverters
 {
 	public static readonly IValueConverter ToFormattedString =
 		new FuncValueConverter<Money?, string>(money => money is null ? "" : money.ToFormattedString());
+
+	public static readonly IValueConverter ToFormattedStringBtc =
```

Comment:
If https://github.com/zkSNACKs/WalletWasabi/pull/8975/files#r961350322 works then remove this function